CHANGELOG.md

```
@@ -1,125 +1,124 @@
 # Prowide Core - CHANGELOG
 
-#### 10.2.13 - SNAPSHOT
+#### 9.5.11 - SNAPSHOT
   * feat: Enhanced SwiftParser for edge cases avoiding OutOfBoundsException when parsing messages with empty blocks
-
-#### 10.2.12 - July 2025
   * (PW-2613) Updated the Field70 codeword splitting logic to allow / as trailing chars
 
-#### 10.2.11 - July 2025
+#### 9.5.10 - July 2025
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)